math/optimized_helper_functions.h

```diff
@@ -0,0 +1,100 @@
+/*
+	Copyright 2021 Kenn Sebesta
+
+	This file is part of the VESC firmware.
+
+	The VESC firmware is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The VESC firmware is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+    */
+
+#ifndef OPTIMIZED_HELPER_FUNCTIONS_H_
+#define OPTIMIZED_HELPER_FUNCTIONS_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+#include "datatypes.h"
+#include <math.h>
+#include <stdlib.h>
```

Comment:
Include only what you need. I think you need `<stdint.h>` and the macros `ONE_BY_SQRT3` and `SIGN`.